exporter/fileexporter/file_exporter.go

```diff
@@ -21,8 +21,8 @@ import (
 
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
-	"github.com/golang/protobuf/jsonpb"
-	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/jsonpb" //lint:ignore SA1019 golang/protobuf/jsonpb is deprecated
+	"github.com/golang/protobuf/proto"  //lint:ignore SA1019 golang/protobuf/proto is deprecated
```

Comment:
Can't replace it with `"google.golang.org/protobuf/proto"` because the OTLP types generated by `gogo/protobuf` don't implement `ProtoReflect()` function of `ProtoMessage` interface:
- https://pkg.go.dev/google.golang.org/protobuf/reflect/protoreflect?tab=doc#ProtoMessage